collections/index.js

```diff
@@ -20,46 +20,130 @@ import employees from './data.json';
   go for it - that's the whole notion of functional programming
 
   The format for this test will be slightly different - the data that I will
-  be using for the test suite lives in ./data.json. I have written you an 
+  be using for the test suite lives in ./data.json. I have written you an
   import statement and you should use these data sets for the test.
 */
+export const howManyEmployees = () => {
+  return employees.length;
+}
 
 /*
   I expect this function to get me an employee by their ID
 */
 export const getEmployeeById = (id) => {
-
+  // created an empty object to insert data into
+  var employeeNumber = {};
```

Comment:
You shouldn't ever really be setting variables with `var` - if it's a variable that's going to change use `let` and if it's a variable that isn't going to change use `const`

Reply:
Also, the variable name `employeeNumber` isn't really representative of what this variable actually contains. Something like `employee` would be more appropriate imo.

Reply:
done